js/src/admin/components/PermissionGrid.js

```diff
@@ -326,12 +326,28 @@ export default class PermissionGrid extends Component {
       60
     );
 
+    items.add('userEditCredentials', {
+      icon: 'fas fa-user-cog',
+      label: app.translator.trans('core.admin.permissions.edit_users_credentials_label'),
+      permission: 'user.editCredentials',
+    });
+
+    items.add(
+      'userEditGroups',
+      {
+        icon: 'fas fa-users-cog',
+        label: app.translator.trans('core.admin.permissions.edit_users_groups_label'),
+        permission: 'user.editGroups',
+      },
+      60
+    );
+
     items.add(
-      'userEdit',
+      'userEditUsername',
       {
-        icon: 'fas fa-user-cog',
-        label: app.translator.trans('core.admin.permissions.edit_users_label'),
-        permission: 'user.edit',
+        icon: 'fas fa-address-card',
+        label: app.translator.trans('core.admin.permissions.edit_users_username_label'),
+        permission: 'user.editUsername',
```

Comment:
The user prefix is required, in my testing without the user. prefix it did not work correctly and always returned false. This is one of those cases of the magic not working correctly.

As for the edit permission I guess it could do the username. TBH I split these permissions based on the previous PR with the only change being the removal of dot notated permissions and upgrading to beta.15.

Reply:
Alternatively, username could also be credentials, and regular edit should be "Other attributes", which would be used by stuff like extensions. Not sure which is better, I'll think about it.

Reply:
I think for right now we might want to make the username a credential. But I think we should have a conversation about using the username for login in the future as I've voiced my opinion on that matter before that it shouldn't be permitted for many reasons. I'll probably create an issue for that later.